magefile.go

```diff
@@ -1012,12 +1012,7 @@ func collectPackageDependencies(platforms []string, packageVersion string, requi
 			// https://artifacts-snapshot.elastic.co/fleet-server/latest/8.11.0-SNAPSHOT.json
 			// https://artifacts-snapshot.elastic.co/prodfiler/latest/8.11.0-SNAPSHOT.json
 			externalBinaries := map[string]string{
-				"auditbeat":             "beats",
-				"filebeat":              "beats",
-				"heartbeat":             "beats",
-				"metricbeat":            "beats",
-				"osquerybeat":           "beats",
-				"packetbeat":            "beats",
```

Comment:
Since we can specify a BEAT_VERSION env var, shouldn't we make a conditional to fetch agentbeat only from 8.14.0 onwards ? If we don't do that packaging will fail for any beat version lower than 8.14.0

Reply:
Each version of elastic-agent is branched so this change will take effect 8.14+ which is where it applies.